hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/UnsupportedOperationVisitor.java

```diff
@@ -145,8 +145,6 @@
         SUPPORTED_KINDS.add(SqlKind.ARGUMENT_ASSIGNMENT);
 
         // Ordering
-        SUPPORTED_KINDS.add(SqlKind.NULLS_FIRST);
```

Comment:
These orderings were supported before, why not just comment these lines and add TODO?

Reply:
It was added after 4.2 before 5.0. No need for TODO unless we decide to support it. It will require a significant change in index structure and index fetching.